src/Workleap.Extensions.Mongo.Abstractions/Workleap.Extensions.Mongo.Abstractions.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net472;net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
for my understanding, is it necessary to have all 3 here? netstandard2.0 should cover both of them

Reply:
Yes it would be enough but you would miss the optimisation specific to some framework.